mail/mpack/files/makefile.am.patch

```diff
@@ -0,0 +1,17 @@
+--- Makefile.am~	2003-07-22 00:39:28
++++ Makefile.am	2025-02-27 18:53:15
+@@ -5,12 +5,12 @@
+ 
+ mpack_SOURCES=unixpk.c encode.c codes.c magic.c unixos.c xmalloc.c \
+ 	md5c.c
+-mpack_LDADD=@LIBOBJS@ @LIB_SOCKET@
++mpack_LDADD=@LIBOBJS@
```

Comment:
What was the error message you got before making this change? Are we sure this is the correct fix?

Reply:
Because the `LIB_SOCKET` wasn't being set at all, even to a null string, the value was being passed on the CLI causing a build failure...

    clang: error: no such file or directory: '@LIB_SOCKET@'

The variable should have been set by a local m4 macro.  The macro wasn't actually being called, but when it is called, it causes more issues as it sets incorrect values for `LIBS` on systems which already have the functions implemented in the standard library.  I've implemented a more theoretically complete fix although the end result is the same but tidier.